SPECv2.md

```diff
@@ -70,9 +70,9 @@ After these a number of fields will exist.
 | Preview secs  | 1        | 'P'   | uint8   | Number of seconds of recording before motion event was detected
 | Latitude      | 4        | 'L'   | float32 | Latitude of device location
 | Longitude     | 4        | 'O'   | float32 | Longitude of device location
-| LocTimestamp  | 8        | 'S'   | uint64  | Time at which location of device was set.  Microseconds since 1970-01-01 UTC
+| LocTimestamp  | 8        | 'S'   | uint64  | Time at which location of device was set.
 | Altitude      | 4        | 'A'   | float32 | Altitude of device location
-| Precision     | 4        | 'R'   | float32 | Estimated precision of location
+| Accuracy      | 4        | 'U'   | float32 | Estimated accuracy of location settings.
```

Comment:
Could you please add "in metres" to the end of the Altitude and Accuracy descriptions?

Reply:
Altitude, yes.  

Accuracy I'm treating as a number between 0 and 1. Like some sort of confidence measure.   Is this actually some measure in metres?

Reply:
Yep, it's actually in metres. That's what the Android location API gives you.

Reply:
Ok I'll change things around a bit.

Reply:
Changed, will require new validation changes to the management-interface now.  Will do that now.